lib/common/formats.js

```diff
@@ -937,10 +938,11 @@ declare const ${moduleName}: ${JSON.stringify(treeWalker(dictionary.tokens), nul
    */
   'ios-swift/enum.swift': function({dictionary, options, file}) {
     const template = _template(
-      fs.readFileSync(__dirname + '/templates/ios-swift/enum.swift.template')
+      fs.readFileSync(__dirname + '/templates/ios-swift/swift.template')
     );
     let allTokens;
     const { outputReferences } = options;
+    file['type'] = 'extension';
```

Comment:
I think this should be `file['type'] = 'enum'`

Reply:
You are totally right, thanks!